mountinfo/mountinfo.go

```diff
@@ -16,15 +22,31 @@ func GetMountsFromReader(reader io.Reader, f FilterFunc) ([]*Info, error) {
 	return parseInfoFile(reader, f)
 }
 
-// Mounted determines if a specified mountpoint has been mounted.
-// On Linux it looks at /proc/self/mountinfo.
-func Mounted(mountpoint string) (bool, error) {
-	entries, err := GetMounts(SingleEntryFilter(mountpoint))
+// Mounted determines if a directory specified by path has been mounted.
+func Mounted(path string) (bool, error) {
+	var st, pst unix.Stat_t
```

Comment:
Looks like we need to split this for the windows case.

Reply:
Right; will do